### ISSUE TITLE
Allow changing the default screen mode

### ISSUE THREAD

**Issue by Rdkang** [jesseduffield/lazydocker#400]
**Is your feature request related to a problem? Please describe.**
Most of the time i do not use the images and volumes panel and is unneeded information for me, So It would be great if i could set the default screen mode.

**Describe the solution you'd like**
An option in `config.yml`:
```yaml 
gui:
   # options should be: normal,half,fullscreen
   screenMode: "half"
   
```
**Describe alternatives you've considered**
An alternative improvement could be being able to toggle panels

**Comment by jesseduffield** [comment]
Works for me. This should be easy to implement: just needs a new config key [here](https://github.com/jesseduffield/lazydocker/blob/b8969fd55f2c446d0fe0f5baf15b2693f1def65b/pkg/config/app_config.go#L132) and then we need to make use of that when the gui is first opened [here](https://github.com/jesseduffield/lazydocker/blob/b8969fd55f2c446d0fe0f5baf15b2693f1def65b/pkg/gui/gui.go#L118) and then we gotta update the config doc with the default value [here](https://github.com/jesseduffield/lazydocker/blob/b8969fd55f2c446d0fe0f5baf15b2693f1def65b/docs/Config.md#L18)

Would you be up to raise a PR for this @Rdkang ?

**Comment by Rdkang** [comment]
Yes, thank you for the helpful links.
I will try to implement it this weekend

> Works for me. This should be easy to implement: just needs a new config key [here](https://github.com/jesseduffield/lazydocker/blob/b8969fd55f2c446d0fe0f5baf15b2693f1def65b/pkg/config/app_config.go#L132) and then we need to make use of that when the gui is first opened [here](https://github.com/jesseduffield/lazydocker/blob/b8969fd55f2c446d0fe0f5baf15b2693f1def65b/pkg/gui/gui.go#L118) and then we gotta update the config doc with the default value [here](https://github.com/jesseduffield/lazydocker/blob/b8969fd55f2c446d0fe0f5baf15b2693f1def65b/docs/Config.md#L18)
> 
> Would you be up to raise a PR for this @Rdkang ?

**Comment by Rdkang** [comment]
Yes, thank you for the helpful links.
I will try to implement it this weekend

**Comment by jesseduffield** [comment]
Awesome, let me know if you run into any issues.

Also I'll re-open this until the issue is complete

**Comment by c0dehu1k** [comment]
gui:
   screenMode: half
# options should also include normal,half,full

what is going to be the behaviour for Normal, Half and Full
Let me try to get this fixed :)

**Comment by Rdkang** [comment]
>  Let me try to get this fixed :)

Thank you @c0dehu1k so much for your interest in helping! I wanted to let you know that I've already implemented the code in #421 :).
I've been using this code for the past six months and it's been working flawlessly. 

Right now, I'm just waiting for @jesseduffield to review and accept the pull request #421.

**Comment by c0dehu1k** [comment]
Super, Thank you @Rdkang for informing me!! 
Let me look for another one ;)
